test/python/primitives/test_backend_estimator.py

```diff
@@ -34,6 +35,18 @@
 BACKENDS = [FakeNairobi(), FakeNairobiV2(), FakeBackendSimple()]
 
 
+class CallbackPass(DummyAP):
```

Comment:
There's not really any need to derive `DummyAP` over the regular `AnalysisPass`, but it also doesn't really hurt.